src/main/java/mate/academy/bookstore/serivce/impl/RoleServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.bookstore.serivce.impl;
+
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.model.Role;
+import mate.academy.bookstore.model.Role.RoleName;
+import mate.academy.bookstore.repository.RoleRepository;
+import mate.academy.bookstore.serivce.RoleService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class RoleServiceImpl implements RoleService {
+    private final RoleRepository roleRepository;
```

Comment:
We don't need role service in this task

Reply:
I thinks it is fine to have it if you want, but just so you know - you could also just use role repository directly in user services and it would be fine